spec/features/1_basic_spec.rb

```diff
@@ -0,0 +1,129 @@
+require "rails_helper"
+
+describe "The /movies page" do
+  it "can be visited", points: 1 do
+    visit "/movies"
+
+    expect(page.status_code).to be(200),
+      "Expected to visit /movies successfully."
+  end
+
+  it "has a link to add a movie", points: 1 do
+    visit "/movies"
+
+    expect(page).to have_link('Add a new movie', href: "/movies/new"),
+      "Expected /movies to have an 'Add a new movie' link to '/movies/new'."
+  end
+
+  it "has a bootstrap navbar", points: 1 do
+    visit "/movies"
+
+    expect(page).to have_selector("nav[class='navbar navbar-expand-lg navbar-light bg-light']"),
```

Comment:
Yes, this could be a problem, but that's why I included what should be there in the "Expected..." hint below. Also if bootstrap changes the order / syntax of the navbar for copy-pasting this test will start to fail. But I think for now we go with it, because the video and lesson content that goes with this matches exactly this class.